server/api/operator.go

```diff
@@ -286,14 +286,19 @@ func (h *operatorHandler) Post(w http.ResponseWriter, r *http.Request) {
 		// support both receiving key ranges or regionIDs
 		startKey, _ := input["start_key"].(string)
 		endKey, _ := input["end_key"].(string)
-		regionIDs, _ := input["region_ids"].([]uint64)
-		group, _ := input["group"].(string)
-		retryLimit, ok := input["retry_limit"].(int)
+		regionIDs, _ := input["region_ids"].([]interface{})
```

Comment:
should it better be `interface{}` ? because `toUint64Slice` accepts `interface{}`.